upgrading/upgrade-ui/1_codegen.md

```diff
@@ -22,7 +22,7 @@ daml codegen js migration-0.1.0.dar -o daml.js
 Once finished, open `ui/package.json`{{open}} in the IDE and add the new dependencies for the
 `create-daml-app-0.1.1`, `forum-0.1.0` and `migration-0.1.0` packages
 
-<pre class="file" data-target="clipboard">
+```
```

Comment:
why was this changed?

Reply:
Sigh, thought this would not get into the PR. I continued working on my master and didn't realise that upstream was set